resources/styles/nheko.qss

```diff
@@ -183,11 +183,25 @@ TopSection {
 
 WelcomePage,
 LoginPage,
+QComboBox,
+QPushButton,
 RegisterPage {
     background-color: white;
     color: #333;
 }
 
+QComboBox:hover,
+QAbstractItemView::item:hover,
+QPushButton:hover {
+    background: #38a3d8;
+    selection-background-color: #38a3d8;
+}
+
+QAbstractItemView::item:selected {
+    border-color: darkblue;
```

Comment:
ah here it is. I don't think this color set goes well with light theme.

makes me think we should have a little block comment at the top of each .qss file that lists all the hex colors used, and what each is for. so when people are adding new styled items, they don't have to come up with new colors to use.

Reply:
Well, my problem is, that I can't style the selected element in comboboxes reasonably at all. I tried that for a pretty long time, but couldn't figure it out. I think I'm just going to set the border color to grey.